README.md

```diff
@@ -255,6 +255,10 @@ tensorboard in inspect mode to inspect the contents of your event files.
 
 ### TensorBoard is showing only some of my data, or isn't properly updating!
 
+> **Update:** After [2.3.0 release][2-3-0], TensorBoard no longer auto uploads
```

Comment:
“uploads” → “updates”? or maybe “reloads”, to be clearer.

Reply:
Changed it to reloads. Thanks for catching an obvious bug